appendices/timers.htm

```diff
@@ -10,7 +10,7 @@
       <li>In-game clock that tracks in-game time - (i.e. time displayed in lower left corner of the game screen) (hereafter referred to as  CINGAME)</li> 
     </ol>
     <br />
-    Time is tracked in ticks; by default a tick is defined as 1/15th of a second. There are 100 frame in a round. A round has 6 AI updates.<br />
+    Time is tracked in ticks; by default a tick is defined as 1/15th of a second. There are 100 frame in a round. A round has 6 AI updates (to be precise, 100/15=6.6 AI updates).<br />
```

Comment:
I don't think this is really useful. 100/6 is not 15, plus pst and iwds have different round lengths (5 and 7).

Reply:
> I don't think this is really useful. 100/6 is not 15, plus pst and iwds have different round lengths (5 and 7).

Got it, removed.